recipes/lua-5.1/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: lua
+  version: 5.1.5
+
+source:
+  fn: lua-5.1.5.tar.gz
+  url: http://www.lua.org/ftp/lua-5.1.5.tar.gz
+  md5: 2e115fe26e435e33b0d5c022e4490567
+  patches:
+    - update.patch  # [not win]
```

Comment:
Please add a description of the patch. Also, we have a policy to send them upstream and follow how that goes here in conda-forge.